core/modules/comment/comment.install

```diff
@@ -470,6 +470,45 @@ function comment_update_1007() {
   }
 }
 
+/**
+ * New setting for comment titles. Replacing comment_subject_field
+ * with comment_title_options.
+ */
+function comment_update_1008() {
+  foreach (config_get_names_with_prefix('node.type.') as $config_name) {
+    $config = config($config_name);
+    $comment_title = $config->get('settings.comment_subject_field');
+    if ($comment_title == COMMENT_TITLE_AUTO) {
```

Comment:
This comparison does not seem correct. `comment_subject_field` has a value of either `TRUE` or `FALSE`, not a string value like `auto`. I think this is intended to do a check like this:

```suggestion
    $comment_title = $config->get('settings.comment_subject_field');
    if ($comment_title == FALSE) {
```